### DISCUSSION TITLE
quick-term should support initial size option in config

### DISCUSSION THREAD

**Discussion by leika** [ghostty-org/ghostty#3876] [Feature Requests, Ideas] [answered]
when switching from iterm or similar, many people have a quickterm-heavy use case. I love the option in ghostty to allow quickterm and keybindings but it would be great if the config would support an initial size in the config so that i can specify a high (fixed or percentage) for top/down terms or a width (percentage or fixed) for left/right positioning.

**Comment by JakeWharton** [comment, marked as answer]
Covered by #2384
